CHANGELOG.md

```diff
@@ -4,7 +4,21 @@ This file summarizes notable changes introduced in aktualizr version. It roughly
 
 Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new release is issued. Thus `N` does not necessarily map to months of the year.
 
-## [??? (unreleased)]
+## [upcoming release]
+
+### Special considerations
+
+As a result of changes to the IP/POSIX Secondary protocol (see below), users of these Secondaries will need to take special care when upgrading their devices. The new version of aktualizr (e.g. the Primary) is backwards compatible and will work with both old and new versions of the protocol. However, the aktualizr-secondary is *not*. This means that if you are upgrading a device with IP/POSIX Secondaries, you should update the Primary ECU first, and if that is successful, then update your Secondaries.
+
+### Added
+
+- SetInstallationRawReport API call to enable setting a custom raw report field in the device installation result: [PR](https://github.com/advancedtelematic/aktualizr/pull/1628)
```

Comment:
I'm not sure about either of these. We added a new API function, and since this is the "Added" section, we aren't strict about complete sentences when it is clear the described feature was added. Neither of the suggestions are accurate, unfortunately.

Reply:
Looks like we can read only this sentence as a continuation to the section heading. It may be a bit misleading to users.

Reply:
Yes, I see your point. But we've been doing that since the beginning and I'm not sure how to make it flow nicely without repeating "Added" in most of the entries.

Reply:
What about this one?
```
- Now, you can use the `SetInstallationRawReport` API call to enable setting a custom raw report field in the device installation result: [PR](https://github.com/advancedtelematic/aktualizr/pull/1628)
```

Reply:
Sure. I took this and simplified it just a bit.